warning messages from python build

Is there anything we can do about these warnings?
```
In file included from /usr/include/python3.4m/Python.h:78:0,
                 from /home/travis/build/GMLC-TDC/HELICS-src/build/swig/python/helicsPYTHON_wrap.c:149:
/usr/include/python3.4m/longobject.h:88:37: warning: ISO C90 does not support ‘long long’ [-Wlong-long]
 PyAPI_FUNC(PyObject *) PyLong_FromLongLong(PY_LONG_LONG);
                                     ^
/usr/include/python3.4m/longobject.h:89:54: warning: ISO C90 does not support ‘long long’ [-Wlong-long]
 PyAPI_FUNC(PyObject *) PyLong_FromUnsignedLongLong(unsigned PY_LONG_LONG);
                                                      ^
/usr/include/python3.4m/longobject.h:90:6: warning: ISO C90 does not support ‘long long’ [-Wlong-long]
 PyAPI_FUNC(PY_LONG_LONG) PyLong_AsLongLong(PyObject *);
      ^
/usr/include/python3.4m/longobject.h:91:15: warning: ISO C90 does not support ‘long long’ [-Wlong-long]
 PyAPI_FUNC(unsigned PY_LONG_LONG) PyLong_AsUnsignedLongLong(PyObject *);
               ^
/usr/include/python3.4m/longobject.h:92:15: warning: ISO C90 does not support ‘long long’ [-Wlong-long]
 PyAPI_FUNC(unsigned PY_LONG_LONG) PyLong_AsUnsignedLongLongMask(PyObject *);
               ^
/usr/include/python3.4m/longobject.h:93:6: warning: ISO C90 does not support ‘long long’ [-Wlong-long]
 PyAPI_FUNC(PY_LONG_LONG) PyLong_AsLongLongAndOverflow(PyObject *, int *);
```

--- a/swig/python/CMakeLists.txt
+++ b/swig/python/CMakeLists.txt
@@ -2,14 +2,7 @@
 IF (GENERATE_PYTHON)
 
 # https://stackoverflow.com/a/3818084/5451769
-if(MSVC)
-  # Force to always compile with W4
-  if(CMAKE_CXX_FLAGS MATCHES "/W[0-4]")
-    # string(REGEX REPLACE "/W[0-4]" "/W4" CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS}")
-  else()
-    # set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /W4")
-  endif()
-elseif(CMAKE_COMPILER_IS_GNUCC OR CMAKE_COMPILER_IS_GNUCXX)
+if(CMAKE_COMPILER_IS_GNUCC OR CMAKE_COMPILER_IS_GNUCXX)
   # Update if necessary
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wno-long-long -pedantic")
 endif()

--- a/swig/python/CMakeLists.txt
+++ b/swig/python/CMakeLists.txt
@@ -1,6 +1,18 @@
 
 IF (GENERATE_PYTHON)
 
+if(MSVC)
+  # Force to always compile with W4
+  if(CMAKE_CXX_FLAGS MATCHES "/W[0-4]")
+    # string(REGEX REPLACE "/W[0-4]" "/W4" CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS}")
+  else()
+    # set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /W4")
+  endif()
+elseif(CMAKE_COMPILER_IS_GNUCC OR CMAKE_COMPILER_IS_GNUCXX)
+  # Update if necessary
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wno-long-long -pedantic")
+endif()
+
 FIND_PACKAGE(PythonLibs REQUIRED)
 INCLUDE_DIRECTORIES(${PYTHON_INCLUDE_PATH})
 INCLUDE_DIRECTORIES(${PYTHON_INCLUDE_DIRS})

--- a/swig/python/CMakeLists.txt
+++ b/swig/python/CMakeLists.txt
@@ -1,6 +1,7 @@
 
 IF (GENERATE_PYTHON)
 
+# https://stackoverflow.com/a/3818084/5451769
 if(MSVC)
   # Force to always compile with W4
   if(CMAKE_CXX_FLAGS MATCHES "/W[0-4]")

--- a/swig/python/CMakeLists.txt
+++ b/swig/python/CMakeLists.txt
@@ -4,7 +4,7 @@ IF (GENERATE_PYTHON)
 # https://stackoverflow.com/a/3818084/5451769
 if(CMAKE_COMPILER_IS_GNUCC OR CMAKE_COMPILER_IS_GNUCXX)
   # Update if necessary
-  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wno-long-long -pedantic")
+  set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wall -Wno-long-long -pedantic")
 endif()
 
 FIND_PACKAGE(PythonLibs REQUIRED)

